### PR TITLE
DEVDOCS-2541: Updated support resources

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-2541](https://jira.bigcommerce.com/browse/DEVDOCS-2541)

## What changed?
* updated partner support resources

### CODE REVIEW

File: docs/api-docs/partner/support.md [hunk 1]
```diff
@@ -8,23 +8,41 @@
...
+
+[BigCommerce Status](https://status.bigcommerce.com/) provides information on the health of BigCommerce-operated services. View [incident history](https://status.bigcommerce.com/history) or subscribe to receive status alerts.
 
 ## Town Hall
+
 [BigCommerce Town Hall](https://support.bigcommerce.com/s/article/BigCommerce-Town-Halls) is a monthly webinar program where merchants and partners receive product and company updates and engage in a live Q&A with BigCommerce executive leadership. Recordings of previous Town Halls are available. 
+
+## Partner Support Team
+
+You can access the Partner Support Team through the Partner Portal. Once you log into your Partner Portal account, hover over the **Support** tab located in the top right corner. Select **Contact Support** and you will be taken to a new page where you can **Open a Support Case** or use our **Live Chat** feature.
```
Comment (traci707) [inline docs/api-docs/partner/support.md:48]: You can access the Partner Support Team through the [Partner Portal](https://partners.bigcommerce.com/). Once you log into your Partner Portal account, hover over the **Support** tab located in the top right corner. Select **Contact Support** and you will be taken to a new page where you can **Create a Support Case** or use our **Live Chat** feature.

File: docs/api-docs/partner/dev-portal-overview.md [hunk 2]
```diff
@@ -21,7 +21,10 @@ For instructions on how to use the Developer Portal to create an app, see [Manag
 
 ### Related articles
 
-* [Introduction to Building Apps](https://developer.bigcommerce.com/api-docs/apps/guide/intro)
+* [BigCommerce APIs Quick Start](https://developer.bigcommerce.com/api-docs/getting-started/making-requests)
+* [BigDesign Developer Playground](https://developer.bigcommerce.com/big-design/)
```
Comment (traci707) [inline docs/api-docs/partner/dev-portal-overview.md:25]: Remove * [BigDesign Developer Playground](https://developer.bigcommerce.com/big-design/) if listed under Tools.

File: docs/api-docs/partner/support.md [hunk 3]
```diff
@@ -8,23 +8,41 @@
...
 
 ## Developer library
+
 The [Developer library](https://developer.bigcommerce.com/dev-library/) provides samples, tools, and guides to help jump-start the perfect ecommerce experience. Find examples and starters to help you begin working on BigCommerce.
 
 ## Developer Slack
-BigCommerce has a dedicated Slack channel for Tech and Agency partners. It is a great place to connect, learn about new updates, and attend “Ask Me Anything” sessions with members of the BigCommerce Product team. To join the BigCommerce developer Slack community, fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSeQApTpd4T9lEX7hPgUjTyoq0vb8UbiCiWseuurJjdMHGBaGQ/viewform). 
+
+BigCommerce has a dedicated Slack channel for Technology and Agency partners. It is a great place to connect, learn about new updates, and attend “Ask Me Anything” sessions with members of the BigCommerce Product team. To join the BigCommerce developer Slack community, fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSeQApTpd4T9lEX7hPgUjTyoq0vb8UbiCiWseuurJjdMHGBaGQ/viewform).
```
Comment (traci707) [inline docs/api-docs/partner/support.md:32]: BigCommerce has a dedicated Slack channel for Technology and Agency partners. It is a great place to connect, learn about new updates, and attend “Ask Me Anything” sessions with the BigCommerce Product team members. To join the BigCommerce developer Slack community, fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLSeQApTpd4T9lEX7hPgUjTyoq0vb8UbiCiWseuurJjdMHGBaGQ/viewform).

**Review by traci707 — CHANGES_REQUESTED** [review]
Found a couple of small items. Good job Katya!

**Comment by ektravel** [comment]
@uwikaiddi This PR is ready for your review.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**